packages/errors/src/codes.ts

```diff
@@ -82,6 +79,12 @@ export const SOLANA_ERROR__KEYS_INVALID_PRIVATE_KEY_BYTE_LENGTH = 3704000 as con
 export const SOLANA_ERROR__KEYS_INVALID_SIGNATURE_BYTE_LENGTH = 3704001 as const;
 export const SOLANA_ERROR__KEYS_SIGNATURE_STRING_LENGTH_OUT_OF_RANGE = 3704002 as const;
 
+// Instruction-related errors.
+// Reserve error codes in the range [4128000-4128999].
+export const SOLANA_ERROR__INSTRUCTION_EXPECTED_TO_HAVE_ACCOUNTS = 4128000 as const;
+export const SOLANA_ERROR__INSTRUCTION_EXPECTED_TO_HAVE_DATA = 4128001 as const;
+export const SOLANA_ERROR__INSTRUCTION_PROGRAM_ID_MISMATCH = 4128002 as const;
+
```

Comment:
Just curious if you have a method for choosing these ranges?

Reply:
Haha I almost asked the same question to @steveluscher before starting this stack.

I just asked my brain to come up with a random number within the range I wanted the range to be in. Maybe Steve's got a more fancy answer though. 😄

Reply:
Just don't mess with my numbering. 😚